src/main/java/com/minecolonies/util/ServerUtils.java

```diff
@@ -105,4 +106,18 @@ public static EntityPlayer getPlayerFromUUID(@Nullable UUID uuid)
         return null;
     }
 
+    public static EntityPlayer getPlayerFromPermPlayer(Permissions.Player player) {
+        return ServerUtils.getPlayerFromUUID(player.getID());
+    }
+
+    public static List<EntityPlayer> getPlayersFromPermPlayer(List<Permissions.Player> players) {
+        List<EntityPlayer> lPlayers = new ArrayList<>();
+
+        for (Permissions.Player player : players) {

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'player' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)